sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfEntrySuite.scala

```diff
@@ -107,7 +107,7 @@ class SQLConfEntrySuite extends SparkFunSuite {
 
   test("stringConf") {
     val key = "spark.sql.SQLConfEntrySuite.string"
-    val confEntry = buildConf(key).stringConf.createWithDefault(null)
+    val confEntry = buildConf(key).stringConf.createWithDefault("")
```

Comment:
I update this because;
```
org.apache.spark.sql.execution.SparkSqlParserSuite.Checks if SET/RESET can accept all the configurations
Error Message
org.scalatest.exceptions.TestFailedException:  == FAIL: Plans do not match ===  SetCommand (spark.sql.SQLConfEntrySuite.string,Some(null))   SetCommand (spark.sql.SQLConfEntrySuite.string,Some(null))
```
https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/126793/testReport/org.apache.spark.sql.execution/SparkSqlParserSuite/Checks_if_SET_RESET_can_accept_all_the_configurations/

Reply:
```
SetCommand (spark.sql.SQLConfEntrySuite.string,Some(null))
SetCommand (spark.sql.SQLConfEntrySuite.string,Some(null))
```
They are exactly the same, not sure why plan check fails...

Reply:
haha, yea, the value in one side is just a string "null" and the value in the other side is actual NULL.